src/modules/dataGridTableRow.js

```diff
@@ -0,0 +1,102 @@
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import React from 'react';
+import Table from '../collections/table.js';
+
+class DataTableRow extends React.PureComponent {
```

Comment:
Same ... should this be `DataGridTableRow`?

Reply:
fixed.